plyer/platforms/ios/flash.py

```diff
@@ -3,7 +3,7 @@
 Flash
 -----
 """
-
+from plyer.facades import Flash
```

Comment:
This should probably be:

    from plyer.facades.flash import Flash

Reply:
I think it's right to do `form plyer.facades import Flash`, because your suggestion is already defined here:

https://github.com/kivy/plyer/blob/6a3d81e5517a8aba7c450d421243c95846d4a2ca/plyer/facades/__init__.py#L26

Reply:
Right you are! :-)